lib/virustotal_api/base.rb

```diff
@@ -50,7 +50,7 @@ def api_uri
 
     # @return [Boolean] if report for resource exists
     def exists?
-      !report.empty?
+      !!report
```

Comment:
What's the deal with the former code? The commit message doesn't really explain why is was buggy and how the new code fixes it.